spec/support/e2e_tests.rb

```diff
@@ -51,6 +51,8 @@ def self.perform(
       # It could be useful to be able to configure this path from the outside (e.g., on CI).
       ::Capybara.save_path = ENV.fetch('CAPYBARA_ARTIFACTS', './tmp/capybara')
 
+      # ::Capybara.disable_animation = true # if this works can we remove the js?
```

Comment:
@ttoomey do you remember if you considered this option before adding the `disable_transitions` code to inject `disable_transitions.js`? Just wondering if maybe the builtin solutions would be sufficient. It's not really related to this PR.

2017 blog post that was referenced https://joshfrankel.me/blog/lessons-learned-from-using-capybara-for-feature-testing/
Builtin animation disabler (seems slightly less comprehensive) https://github.com/teamcapybara/capybara/blob/0480f90168a40780d1398c75031a255c1819dce8/lib/capybara/server/animation_disabler.rb#L64-L77

Reply:
I don't recall but let's give it a shot; it would be nice to replace it with the built-in. Thanks!